ballsdex/packages/players/cog.py

```diff
@@ -134,6 +159,240 @@ async def delete(self, interaction: discord.Interaction):
+                f"You are now friends with {user.name}.", ephemeral=True
+            )
+
+    @friend.command(name="remove")
+    async def friend_remove(self, interaction: discord.Interaction, user: discord.User):
+        """
+        Remove a friend.
+
+        Parameters
+        ----------
+        user: discord.User
+            The user you want to remove as a friend.
+        """
+        player1, _ = await PlayerModel.get_or_create(discord_id=interaction.user.id)
+        player2, _ = await PlayerModel.get_or_create(discord_id=user.id)
+
+        if player1 == player2:
+            await interaction.response.send_message("You cannot remove yourself.", ephemeral=True)
+            return
+        if user.bot:
+            await interaction.response.send_message("You cannot remove a bot.", ephemeral=True)
+            return
+
+        friendship_exists = await player1.is_friend(player2)
+        if not friendship_exists:
+            await interaction.response.send_message(
+                "You are not friends with this user.", ephemeral=True
+            )
+            return
+        else:
+            await Friendship.filter(
+                (Q(player1=player1) & Q(player2=player2))
+                | (Q(player1=player2) & Q(player2=player1))
+            ).delete()
+            await interaction.response.send_message(
+                f"{user.name} has been removed as a friend.", ephemeral=True
+            )
+
+    @friend.command(name="list")
+    async def friend_list(self, interaction: discord.Interaction["BallsDexBot"]):
+        """
+        View all your friends.
+        """
+        player, _ = await PlayerModel.get_or_create(discord_id=interaction.user.id)
+
+        friendships = (
+            await Friendship.filter(Q(player1=player) | Q(player2=player))
+            .select_related("player1", "player2")
+            .all()
+        )
+
+        if not friendships:
+            await interaction.response.send_message("You haven't got any friends!", ephemeral=True)
+            return
+
+        entries: list[tuple[str, str]] = []
+
+        for idx, relation in enumerate(friendships, start=1):
+            if relation.player1 == player:
+                friend = relation.player2
+            else:
+                friend = relation.player1
+
+            since = format_dt(relation.since, style="f")
+            entries.append((f"{idx}. {friend.discord_id}\n", f"Since: {since}"))
+
+        source = FieldPageSource(entries, per_page=5, inline=False)
+        source.embed.title = "Friend List"
+        source.embed.set_thumbnail(url=interaction.user.display_avatar.url)
+        source.embed.set_footer(text="To add a friend, use the command /player friends add.")
+
+        pages = Pages(source=source, interaction=interaction, compact=True)
+        await pages.start(ephemeral=True)
+
+    @app_commands.command()
+    async def block(self, interaction: discord.Interaction, user: discord.User):
```

Comment:
```suggestion
    @blocked.command(name="add")
    async def block_add(self, interaction: discord.Interaction, user: discord.User):
```

Should be using the block subcommand here